tests/integration/targets/mso_schema_site_external_epg/tasks/main.yml

```diff
@@ -961,7 +969,7 @@
       - nm_no_site_associated is not changed
       - cm_no_site_associated.msg == nm_no_site_associated.msg == "Provided site 'ansible_test' not associated with template 'Template4'. Site is currently associated with template(s){{':'}} Template1, Template2, Template3"
 
-# Verify route_reachability argument when template_external_epg is associated with Azure site and
+# Verify route_reachability argument when template_external_epg is associated with ansible_test site and
 # template_external_epg type argument is set to cloud
 - name: Ensure ANP exists
   cisco.mso.mso_schema_template_anp:
```

Comment:
Remove from L972 to L1061 as route_reachability is a Cloud only feature.

Reply:
Done